molecule/python/tests/test_python.py

```diff
@@ -18,6 +18,7 @@
         "/tools/sshenum",
         "/tools/dirsearch",
         "/tools/mitm6",
+        "/tools/sqlmap",
```

Comment:
If it doesn't hurt anything to alphabetize.

```suggestion
        "/tools/dirsearch",
        "/tools/mitm6",
        "/tools/sqlmap",
        "/tools/sshenum",
```